libs/SalesforceSDK/src/com/salesforce/androidsdk/push/PushService.java

```diff
@@ -179,25 +142,6 @@ private void performRegistrationChange(boolean register, UserAccount userAccount
 		}
 	}
 
-    private void scheduleSFDCRegistrationRetry(long when, UserAccount account) {
```

Comment:
This is replaced with a simple parameter in `enqueuePushNotificationsRegistrationWork`.